src/main/java/com/github/manolo8/darkbot/gui/drawables/ConstantEntitiesDrawer.java

```diff
@@ -0,0 +1,71 @@
+package com.github.manolo8.darkbot.gui.drawables;
+
+import com.github.manolo8.darkbot.extensions.features.Feature;
+import eu.darkbot.api.extensions.Drawable;
+import eu.darkbot.api.extensions.MapGraphics;
+import eu.darkbot.api.game.entities.BattleStation;
+import eu.darkbot.api.game.entities.Portal;
+import eu.darkbot.api.game.entities.Station;
+import eu.darkbot.api.managers.EntitiesAPI;
+
+import java.util.Collection;
+
+@Feature(name = "Constant Entities Drawer", description = "Draws not-movable entities (eg: portals, CBS, and stations)")
```

Comment:
```suggestion
@Feature(name = "Constant Entities Drawer", description = "Draws entities that are constantly on the map (eg: portals, CBS, and stations)")
```
refering to *non-movable* may be misleading. eg: mines don't move, but are not drawn by this. I think the more relevant "distinction" is that the entities in here are always on the map, while others may come and go as you get closer (usually)